operations/finance.md

```diff
@@ -257,7 +257,10 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
     "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
     "Client": "Sample Client 1.0.0",
     "StartUtc": "2017-01-01T00:00:00Z",
-    "EndUtc": "2017-02-01T00:00:00Z"
+    "EndUtc": "2017-02-01T00:00:00Z",
+    "ItemIds": [
```

Comment:
With this filter, the example response doesn't make sense. I'd remove ItemIds from this example request.